docs/en/sql-reference/statements/drop.md

```diff
@@ -30,7 +30,7 @@ Also see [UNDROP TABLE](/docs/en/sql-reference/statements/undrop.md)
 Syntax:
 
 ``` sql
-DROP [TEMPORARY] TABLE [IF EXISTS] [IF EMPTY] [db.]name [ON CLUSTER cluster] [SYNC]
+DROP [TEMPORARY] TABLE [IF EXISTS] [IF EMPTY] [db.]name_1[,[db.]name_2][,[db.]name_3]... [ON CLUSTER cluster] [SYNC]
```

Comment:
would be nice to support multi-dropping also for users, roles, etc.
doesn't have to be in this PR but it would be nice to open an issue for this.

Reply:
Ok, I will try this function.